File: lab3/src/main/java/lab3/myquest/locations/Floor.java
package lab3.myquest.locations;

import lab3.functionality.Location;
import lab3.functionality.Player;

public class Floor extends Location {
    public Floor(String name) {
        super(name, "Очень твердый пол");
    }

    @Override
    public String enter(Player player) {
        return player.getName() + " упал на пол, чувствуя, что его как бы подхватило ветром и куда-то несет.";
    }
}

File: lab4/src/main/kotlin/lab4/myQuest/locations/Floor.kt
package lab4.myQuest.locations

import lab4.functionality.Location
import lab4.functionality.Player
import lab4.myQuest.items.Chair

class Floor : Location("Пол", { it.name + " лежит на полу. Пол очень твёрдый" }) {
    init {
        items.add(Chair())
    }

    override fun enter(player: Player): String {
        return "${player.name} упал на пол, чувствуя, что его как бы подхватило ветром и куда-то несет."
    }
}